tests/testthat/test-colour_nhsbsa.R

```diff
@@ -0,0 +1,12 @@
+library(NHSRtheme)
+
+testthat::test_that("custom colour palette returns lengh > 1",{
+  expect_equal(length(
+    palette_nhsbsa(palette = NHSRtheme::get_nhs_colours(c("Blue","Yellow","Pink"))
+    )),3)
+
+  # custom colour should have more than one colour
+  # If you want to use one colour then default NA will render
+  expect_error(palette_nhsbsa(palette = NHSRtheme::get_nhs_colours("Pink")))
+
+})
```

Comment:
Could add further tests, ideally want to test each route through the function, like I did informally in above comment.